icgcget/cli.py

```diff
@@ -26,9 +26,10 @@
 from clients.gdc import gdc_client
 from clients.gnos import gt_client
 from clients.icgc import icgc_client
-from utils import flatten_dict, normalize_keys
+from clients.icgc import icgc_api
+from utils import flatten_dict, normalize_keys, match_repositories
 
-REPOS = ['collab', 'aws', 'ega', 'gdc', 'cghub']
+REPOS = ['collaboratory', 'aws-virginia', 'ega', 'gdc', 'cghub']  # Updated for codes used by api
```

Comment:
I would still use the old values as they are shorter and easier to type. Also, we use them in the config / options settings. The values above should be encapsulated in your `icgc_api` module (probably via a dict) and not exposed in this file.